frontend/src/components/eMIB/EditTask.jsx

```diff
@@ -69,8 +69,12 @@ const styles = {
 
 class EditTask extends Component {
   state = {
-    task: !this.props.action ? "" : this.props.action.task,
-    reasonsForAction: !this.props.action ? "" : this.props.action.reasonsForAction
+    task: !this.props.action ? "" : !this.props.action.task ? "" : this.props.action.task,
```

Comment:
Can we add a test to show that this component doesn't show an error if we are in this case?

Reply:
Added in 2b76bf8